clients/drcachesim/CMakeLists.txt

```diff
@@ -333,7 +333,15 @@ endif ()
 # We build larger executables here.  All tests are added in suite/tests/.
 # Be sure to give the targets qualified test names ("tool.drcache*...").
 
+
 if (BUILD_TESTS)
+  add_executable(tool.drcachesim.unit_tests tests/drcachesim_unit_tests.cpp)
+  if (ZLIB_FOUND)
+    target_link_libraries(tool.drcachesim.unit_tests simulator drmemtrace_static ${ZLIB_LIBRARIES})
+  else ()
+    target_link_libraries(tool.drcachesim.unit_tests simulator drmemtrace_static)
+  endif ()
+  add_win32_flags(tool.drcachesim.unit_tests)
```

Comment:
Wait -- there's no add_test!  How was it tested locally?  Please add and verify the test runs locally (e.g. `ctest -R sim.unit`) and is run in the CI.

Reply:
Done. Link to CI

https://travis-ci.org/DynamoRIO/dynamorio/jobs/316666701#L809